app/src/models.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+from sqlalchemy import Enum as SQLAlchemyEnum
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import CheckConstraint
+from datetime import datetime, timezone, timedelta
+from flask_bcrypt import Bcrypt
+
+bcrypt = Bcrypt()
+db = SQLAlchemy()
+
+# Predefined grade selections
+# Note: when adding a new user, use "grade=GradeEnum.NINTH" to assign a grade (in this example, 9th grade)
+class GradeEnum(Enum):
+    FIRST = '1st Grade'
+    SECOND = '2nd Grade'
+    THIRD = '3rd Grade'
+    FOURTH = '4th Grade'
+    FIFTH = '5th Grade'
+    SIXTH = '6th Grade'
+    SEVENTH = '7th Grade'
+    EIGHTH = '8th Grade'
+    NINTH = '9th Grade'
+    TENTH = '10th Grade'
+    ELEVENTH = '11th Grade'
+    TWELFTH = '12th Grade'
+    FRESHMAN = 'University Freshman'
+    SOPHMORE = 'University Sophmore'
+    JUNIOR = 'University Junior'
+    SENIOR = 'University Senior'
+
+# User model
+class User(db.Model):
```

Comment:
Generally it may be better to not impose too small of a string length, higher than 150 is preferred for things like Name and Email